fastapi-app/models/recipe_ingredient.py

```diff
@@ -0,0 +1,20 @@
+import uuid
+
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy import Column, String, DateTime, ForeignKey
+
+from db.base_class import Base, UtcNow
+
+
+class recipe_ingredient(Base):
```

Comment:
Class name SHOULD start with capital letter and follow the CamelCase style of name declaration, please correct this.